js.js

```diff
@@ -0,0 +1,40 @@
+const multiplicate = document.getElementById("button");
+const input = document.getElementById("input");
+const table = document.getElementById("table");
+const form = document.getElementById("form");
+let result = 1;
+
+form.addEventListener("submit", (e) => {
+    e.preventDefault();
+})
```

Comment:
Isto é apenas preferências, mas quando as funções são assim pequenas eu gosto de meter tudo na mesma linha:

```suggestion
form.addEventListener("submit", e => e.preventDefault());
```

E se quiseres até podes fazer assim, para simplificar:

```suggestion
document.getElementById("form").addEventListener("submit", e => e.preventDefault());
```